engine/src/flutter/impeller/compiler/spirv_compiler.cc

```diff
@@ -282,6 +282,7 @@ shaderc::CompileOptions SPIRVCompilerOptions::BuildShadercOptions() const {
 
   options.SetAutoBindUniforms(true);
   options.SetAutoMapLocations(true);
+  options.SetPreserveBindings(true);
```

Comment:
Try setting this to true only when this is one of the "runtime stage" modes (runtime-stage-vulkan, et cetera) those are the developer authored shaders

Reply:
I was hoping to get this working for us too since it makes working on the shaders a pain.